### PR TITLE
Port helm code

### PR DESCRIPTION
**Pull request by oraNod**
Following on from the [announcement to relocate Helm chart code](https://forum.ansible.com/t/upcoming-changes-to-awx-operator-installation-methods/7598) for the AWX Operator from the `ansible/awx-operator` repo, this PR moves files related to the Helm chart and adds a script to clone relevant parts of the repo so that it is possible to generate the Helm chart with the Makefile.

### CODE REVIEW

**Comment by oraNod** [comment]
I've converted to draft so we don't merge this from the UI, which could mess up the git history. When we're ready to merge, use `git merge --ff` to get this branch into main and then push it from CLI.

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,449 @@
+# VERSION defines the project version for the bundle.
```
Comment (TheRealHaoLiu) [inline Makefile:1]: only need to move the helm related make targets and the variables that those make targets use 

when we clone down awx-operator we can rename awx-operator's makefile as Makefile.awx-operator
than at the top of this Makefile do a include to combine the 2 Makefile

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+endif
+
+charts:
+	mkdir -p $@
+
+.PHONY: helm-chart
+helm-chart: helm-chart-generate
+
+.PHONY: helm-chart-generate
+helm-chart-generate: kustomize helm kubectl-slice yq charts
```
Comment (TheRealHaoLiu) [inline Makefile:62]: need to copy over the following make targets
kubectl-slice, yq and kustomize

Comment (oraNod) [reply Makefile:111]: @TheRealHaoLiu Should be all set now. Cheers.

**Review by TheRealHaoLiu — APPROVED** [review]

**Pull request merged**